### PR TITLE
Mention the signed extensions in the reference docs

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

File: docs/sdk/src/reference_docs/signed_extensions.rs [hunk 1]
```diff
@@ -1,7 +1,51 @@
...
+//! - [`ChargeAssetTxPayment`(using
+//!   conversion)](pallet_asset_conversion_tx_payment::ChargeAssetTxPayment): Charges transaction
+//!   fees from the signer based on the weight of the call using any supported asset (including the
+//!   native token). The asset is converted to the native token using a pool.
+//!
+//! - [`SkipCheckIfFeeless`](pallet_skip_feeless_payment::SkipCheckIfFeeless): Allows transactions
+//!   to be processed without paying any fee. This requires that the `call` that should be
+//!   dispatched is augmented with the
+//!   [`feeless_if`](macro@frame_support::pallet_macros::feeless_if) attribute.
+//!
```
Comment (sandreim) [inline docs/sdk/src/reference_docs/signed_extensions.rs:45]: `CheckMetadataHash` missing ?

**Review by sandreim — APPROVED** [review]

File: docs/sdk/src/reference_docs/signed_extensions.rs [hunk 2]
```diff
@@ -1,7 +1,51 @@
 //! Signed extensions are, briefly, a means for different chains to extend the "basic" extrinsic
 //! format with custom data that can be checked by the runtime.
 //!
-//! # Example
+//! # FRAME provided signed extensions
+//!
+//! FRAME by default already provides the following signed extensions:
+//!
+//! - [`CheckGenesis`](frame_system::CheckGenesis): Ensures that a transaction was send for the same
```
Comment (gupnik) [inline docs/sdk/src/reference_docs/signed_extensions.rs:8]:
```suggestion
//! - [`CheckGenesis`](frame_system::CheckGenesis): Ensures that a transaction was sent for the same
```

File: docs/sdk/src/reference_docs/signed_extensions.rs [hunk 3]
```diff
@@ -1,7 +1,51 @@
...
+//! - [`CheckMortality`](frame_system::CheckMortality): Extends a transaction with a configurable
+//!   mortality.
+//!
+//! - [`CheckNonZeroSender`](frame_system::CheckNonZeroSender): Ensures that the sender of a
+//!   transaction is not the *all zero account* (all bytes of the accountid are zero).
+//!
+//! - [`CheckNonce`](frame_system::CheckNonce): Extends a transaction with a nonce to prevent replay
+//!   of transactions and to provide ordering of transactions.
+//!
+//! - [`CheckSpecVersion`](frame_system::CheckSpecVersion): Ensures that a transaction was build for
```
Comment (gupnik) [inline docs/sdk/src/reference_docs/signed_extensions.rs:20]:
```suggestion
//! - [`CheckSpecVersion`](frame_system::CheckSpecVersion): Ensures that a transaction was built for
```

**Review by gupnik — APPROVED** [review]

File: docs/sdk/src/reference_docs/signed_extensions.rs (hunk 1 above)
Comment (skunert) [reply docs/sdk/src/reference_docs/signed_extensions.rs:45]: Probably also worth mentioning `StorageWeightReclaim` as the target audience of these docs are mostly parachain developers.

**Review by skunert — APPROVED** [review]

**Pull request merged**